Source/Core/AudioCommon/Mixer.cpp

```diff
@@ -373,3 +374,22 @@ unsigned int Mixer::MixerFifo::AvailableSamples() const
     return 0;  // Mixer::MixerFifo::Mix always keeps one sample in the buffer.
   return (samples_in_fifo - 1) * m_mixer->m_sampleRate / m_input_sample_rate;
 }
+
+u32 Mixer::DPL2QualityToFrameBlockSize(int quality)
```

Comment:
This can be an internal member function placed within an anonymous namespace above the Mixer constructor. `quality` should likely also be an enum class, to void using magic numbers as case values.